_tutorials/2016-05-10-iosxr-ansible.md

```diff
@@ -136,10 +139,11 @@ vagrant@localhost's password:
 RP/0/RP0/CPU0:ios#
 ```
 
-Now let's  configure IP address at XR box. Issue the commant at XR CLI.   
+Now let's configure an IP address on the IOS-XRv instance. Issue the following command on the XR cli:
 
```

Comment:
Do you want to try the Management network and use the redirected port instead, as we discussed? Then you can skip the XR config section.